index.html

```diff
@@ -37,7 +37,7 @@
     <h4 class="text-center mt-5">Handcrafted, home-made masterpieces</h4>
     <form class="form-inline mx-5" style="width:20rem ;">
       <input class="" type="search" placeholder="Enter your email address" aria-label="Search">
-      <button class="btn" type="submit">Get Started</button>
+      <button class="btn" type="submit">Let's Get Started</button>
```

Comment:
- [ ]As required for this project you should wrote at least 2 comments in the review. kindly write at least 2 comments and be specific, explain what you want the student to change or to improve :upside_down_face: